priv/lib-src/elm/src/SearchPage.elm

```diff
@@ -71,10 +74,25 @@ init flags =
             Decode.decodeValue Flags.fromJson flags
                 |> Result.withDefault Flags.defaultFlags
 
-        { filters, language, screenWidth, excludeCategories, queryString, pageLength } =
+        { language, screenWidth, excludeCategories, queryString, pageLength } =
             decodedFlags
+
+        ( preparedFilters, filterEffects ) =
+            initializeFilters decodedFlags.filters
+
+        initialCommands =
+            filterEffects
+                |> List.filterMap filterEffectToCmd
+
+        initialCmd =
+            case initialCommands of
+                [] ->
+                    Cmd.none
+
+                _ ->
+                    Cmd.batch initialCommands
```

Comment:
Nitpick, but [as per documentation](https://package.elm-lang.org/packages/elm/core/latest/Platform.Cmd):
> Note: `Cmd.none` and `Cmd.batch [ Cmd.none, Cmd.none ]` and `Cmd.batch []` all do the same thing.

so I think it's unnecessary for this differentiation, this could simply be:
```suggestion
        initialCmd = Cmd.batch initialCommands
```